lcidlc/lclink.sh

```diff
@@ -1,12 +1,17 @@
 # Dependencies are calculated from the .ios file that lists which libraries and frameworks to link
 # against.
-if [ "$LIVECODE_DEP_FILE" == "" ]; then
-    LIVECODE_DEP_FILE="$SRCROOT/$PRODUCT_NAME.ios"
-fi
 
 read SDK_MAJORVERSION SDK_MINORVERSION <<<${SDK_NAME//[^0-9]/ }
 read SDK_PLATFORM <<<${SDK_NAME//[0-9.]/ }
 
+if [ "$LIVECODE_DEP_FILE" == "" ]; then
```

Comment:
@livecodepanos sorry. Little late to the review. This change is not necessary and should be reverted because lclidl based externals have a different mechanism for conditional inclusion of frameworks